game/src/OptionMenu/VideoTab.tscn

```diff
@@ -40,9 +40,21 @@ selected = 0
 popup/item_0/text = "MISSING"
 popup/item_0/id = 0
 script = ExtResource("1_i8nro")
+revert_dialog = NodePath("ConfirmationDialog")
+timer = NodePath("Timer")
 section_name = "video"
 setting_name = "resolution"
 
+[node name="ConfirmationDialog" type="ConfirmationDialog" parent="VBoxContainer/GridContainer/ResolutionSelector"]
+editor_description = "UI-873"
+disable_3d = true
+title = "OPTIONS_VIDEO_RESOLUTION"
```

Comment:
Prefer separate titles for dialogs in menus.csv, if you prefer the title not overflow, please keep it really short so it likely does not overflow please. Also I would recommend you use `OPTIONS_VIDEO_RESOLUTION_DIALOG_TITLE` for that.